platform/util/platform-util/pom.xml

```diff
@@ -70,6 +70,12 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
+            <artifactId>system-rules</artifactId>
+            <version>1.16.1</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I took a look. 

This dependency exists @`/platform/security/sts/security-sts-ldaplogin/pom.xml`. Are there drawbacks on moving it to a common `pom.xml`?